scripts/mock-upcoming-ipodata.json

```diff
@@ -22,5 +22,11 @@
         "IpoFullName": "Bharti Upcoming Hexacom IPO-4",
         "IPOEventStartDate": "03 Apr 2024",
         "IPOEventEndDate": "05 Apr 2024"
+    },
+    {
+        "IPOLogoImage": "https://www.icicidirect.com/images//Bharti Hexacom Logo-202403271411036233854.png",
```

Comment:
@ravverma  this would be remove once merge and will be used for original pr ?

Reply:
Keeping it to show-n-tell as current api returns only one, Once showed to customer will revert back to API

Reply:
I'd suggest moving to feature branch and do Shon-n-tell from there.

Reply:
@sdmcraft  Agreed, will use this branch for demo purpose.

Reply:
@ravverma  IMO then we can close this PR

Reply:
Yes, let me close this.